staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -101,6 +101,9 @@ func (f *FieldManager) Update(liveObj, newObj runtime.Object, manager string) (r
 		if err != nil {
 			return nil, fmt.Errorf("failed to decode managed fields: %v", err)
 		}
+                if len(managed) == 0 {
```

Comment:
This is not going to work. It's ok not to have managedfields, when an object is created for example. We do check old and new object to allow people to "manually change" the manual fields, but still fallback in case it's completely broken.

Reply:
We have a nice comment explaining above, maybe we could clarify why it's fine to have an empty managedFields?